src/libutil/filter.cpp

```diff
@@ -230,9 +230,10 @@ class FilterCatmullRom1D : public Filter1D {
         x = fabsf(x);
         float x2 = x * x;
         float x3 = x * x2;
-        return (x >= 2.0f) ? 0.0f :  ((x < 1.0f) ?
+        float y = (x >= 2.0f) ? 0.0f :  ((x < 1.0f) ?
```

Comment:
You know, I think I'm going to try another approach with catrom. I'm going to revert that to the original (widely published) formula, and merely change my graphing approach to rescale it